{"failedTests":[{"testName":"parallel/test-without-async-context-frame","timestamp":1736788573251,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win2019-COMPILED_BY-vs2022 #32051","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2019-COMPILED_BY-vs2022/32051/","buildNumber":32051,"builtOn":"test-rackspace-win2019_vs2019-x64-2","nodeVersion":"24.0.0","commitHash":"27746115311f9e724a6064656838582192b400a6","callStack":[{"upstreamBuild":67825,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":76998,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":64479,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"56559"}],"tap":{"ok":false,"name":"parallel/test-without-async-context-frame","id":992,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ without AsyncContextFrame\n  ✖ async-hooks\\test-async-local-storage-args.js (4372.2269ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-args.js failed with exit code 3221225477\n    \n    3221225477 !== 0\n    \n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-without-async-context-frame.mjs:59:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:921:9)\n        at async Promise.all (index 0)\n        at async Suite.run (node:internal/test_runner/test:1300:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:297:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-async-await.js (1624.1968ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (1917.9365ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (1295.0928ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (1321.8907ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (1423.2873ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (1418.5407ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (1263.0281ms)\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (1299.1411ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (1279.0769ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (1278.5841ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (1286.4725ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (1606.4396ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (1245.9534ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (1286.8075ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (1201.6741ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (1552.6436ms)\n  ✔ parallel\\test-async-local-storage-bind.js (1369.0399ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (1337.2592ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (1246.5785ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (1223.2873ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (1501.6101ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (1136.2438ms)\n✖ without AsyncContextFrame (34509.4424ms)\nℹ tests 23\nℹ suites 1\nℹ pass 22\nℹ fail 1\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 34524.1684\n\n✖ failing tests:\n\ntest at test\\parallel\\test-without-async-context-frame.mjs:49:5\n✖ async-hooks\\test-async-local-storage-args.js (4372.2269ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-args.js failed with exit code 3221225477\n  \n  3221225477 !== 0\n  \n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-without-async-context-frame.mjs:59:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:921:9)\n      at async Promise.all (index 0)\n      at async Suite.run (node:internal/test_runner/test:1300:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:297:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":35050.157,"fullname":"parallel/test-without-async-context-frame","closingTestPoint":false}}]}